packages/aragon-wrapper/src/core/proxy/index.js

```diff
@@ -1,6 +1,21 @@
 import { fromEvent, from } from 'rxjs'
 import { filter } from 'rxjs/operators'
 
+function getEventNames (eventNames) {
+  // Get all events
+  if (!eventNames) {
+    eventNames = ['allEvents']
```

Comment:
What do you think about just returning rather than assigning a new value:
```suggestion
    return ['allEvents']
```